app/views.py

```diff
@@ -152,8 +135,8 @@ def add_a_list():
 									})
 				return response, 201
 			return jsonify({'status': 'fail', 'message': 'list is missing in the data'}), 400
-		return jsonify({"error":"user not in isinstance"}), 401
-	return jsonify({"error":" cant access to login"}), 401
+		return jsonify({'status': 'fail', 'message':'user not in isinstance'}), 401

```

Comment:
"user not in instance" is not a very helpful message, for a user.. 401 being the status code, isnt it something like they are not authorized?

Reply:
worked on it